src/test/java/lawrence/parser/CommandParserTest.java

```diff
@@ -0,0 +1,138 @@
+package lawrence.parser;
+
+import lawrence.command.Command;
+import lawrence.command.DisplayTasksCommand;
+import lawrence.command.ExitSessionCommand;
+import lawrence.command.CompleteTaskCommand;
+import lawrence.command.UncompleteTaskCommand;
+import lawrence.command.DeleteTaskCommand;
+import lawrence.command.AddTaskCommand;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CommandParserTest {
```

Comment:
Great job on considering multiple exceptions that could occur! Very extensive test case!